coderd/database/migrations/000094_app_usage.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS app_usage (
+    user_id uuid NOT NULL REFERENCES users (id) ON DELETE CASCADE,
+    app_id uuid NOT NULL REFERENCES workspace_apps (id) ON DELETE CASCADE,
+    template_id uuid NOT NULL REFERENCES templates (id) ON DELETE CASCADE,
+	created_at DATE NOT NULL,
+	UNIQUE (user_id, app_id, template_id, created_at)
```

Comment:
mixed tabs and spaces

Reply:
Not sure if I got it.

Reply:
These two lines seem to use tab characters instead of the other lines which use 4 spaces. You can see it easily on github.com if you look at the file in the diff